index.html

```diff
@@ -0,0 +1,215 @@
+                </div>
+
+                <div class="card">
+                    <p class="headline-text">We aim to shape possibilities of
+                        under privileged
+                        children by providing a stepping stone to acquiring
+                        education and better living conditions.</p>
+                </div>
+
+                <q>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </q>
+
+            </section>
+
+        </div>
+        <section class="values">
+            <h3>Our Values</h3>
+            <hr>
+            <div class="values-container">
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li><img
+                                src="/first-capstone-project/assets/images/integrity.png" /></li>
+                        <li><p>Integrity</p>
+                        </li>
+                        <li>
+                            <p>We focus on the entire customer lifecycle.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="values-reasons" id="values">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/responsive.png" />
+                            <li><p>Responsive</p></li>
+                            <li>
+                                <p>To the needs of the children in our target
+                                    group
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/enriching.png" />
+                            <li><p>Enriching</p></li>
+                            <li>
+                                <p>
+                                    bettering lives through education and
+                                    other avenues
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+            </div>
+            <button type="button" class="donate-btn">Donate</button>
+
+            <div class="donate">
+                <img src="/first-capstone-project/assets/images/donate.png" />
+            </div>
+
+        </section>
+        <section class="projects">
+            <h3>Upcoming Events</h3>
+            <hr class="title-hr" />
+            <div class="events-list"></div>
+            <div class="more-btn" id="more-btn">
+                <p class="more">MORE</p>
+                <i class="arrow down"></i>
```

Comment:
Well done by implementing the more and less feature but on mobile when I click the button nothing happens please make sure it is working and if you have problems to make work don't hesitate to ask for help 

![Capture d’écran du 2023-08-02 00-52-18](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/f2c2a6dc-02ca-4c99-9f87-764eec1078da)